theano/tensor/sort.py

```diff
@@ -450,7 +459,7 @@ def topk(x, k, axis=-1, idx_dtype='int64'):
 
     Notes
     -----
-    - The returned values may not be sorted.
+    - The returned values are not sorted.
```

Comment:
I took a look at PyTorch and TF, both gives sorted result by default. Didn't check out other frameworks.
Could be a good idea to finish GPU sort/argsort first.